addons/event/models/event_mail.py

```diff
@@ -114,13 +119,48 @@ def execute(self):
                 self.write({'mail_sent': True})
         return True
 
+    @api.model
+    def _warn_template_error(self, scheduler, exception):
+        if random.random() < 0.15:
+            ex_s = exception_to_unicode(exception)
+            try:
+                event, template = scheduler.event_id, scheduler.template_id
+                emails = list(set([event.organizer_id.email, event.user_id.email, template.write_uid.email]))
+                subject = _("WARNING: Event Scheduler Error for event: %s" % event.name)
+                body = _("""Event Scheduler for:
+                              - Event: %s (%s)
+                              - Planified: %s
+                              - Template: %s (%s)
+                            \nFailed with error:
+                              - %s
+                            \nYou receive this email because you are:
+                            - the organizer of the event,
+                            - or the responsible of the event,
+                            - or the last writer of the template."""
+                         % (event.name, event.id, scheduler.scheduled_date, template.name, template.id, ex_s))
+                email = self.env['ir.mail_server'].build_email(
+                    email_from=self.env.user.email,
+                    email_to=emails,
+                    subject=subject, body=body,
+                )
+                self.env['ir.mail_server'].send_email(email)
+            except Exception as e:
+                _logger.error("Exception while sending traceback by email: %s.\n Original Traceback:\n%s", e, exception)
+                pass
+
     @api.model
     def run(self, autocommit=False):
         schedulers = self.search([('done', '=', False), ('scheduled_date', '<=', datetime.strftime(fields.datetime.now(), tools.DEFAULT_SERVER_DATETIME_FORMAT))])
         for scheduler in schedulers:
-            scheduler.execute()
-            if autocommit:
-                self.env.cr.commit()
+            try:
+                with self.env.cr.savepoint():
+                    scheduler.execute()
+            except Exception as e:
```

Comment:
Is the error already logged at this point? We should always have it in the logs.